src/community/wps-jdbc/src/test/java/org/geoserver/wps/jdbc/JDBCStatusStoreTest.java

```diff
@@ -0,0 +1,137 @@
+package org.geoserver.wps.jdbc;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+
+import org.geoserver.wps.AbstractProcessStoreTest;
+import org.geoserver.wps.ProcessStatusStore;
+import org.geoserver.wps.executor.ExecutionStatus;
+import org.geotools.data.DataStore;
+import org.geotools.data.DataStoreFinder;
+import org.geotools.data.postgis.PostgisNGDataStoreFactory;
+import org.geotools.feature.NameImpl;
+import org.junit.After;
+import org.junit.Test;
+import org.opengis.filter.Filter;
+
+import net.opengis.ows11.CodeType;
+import net.opengis.ows11.Ows11Factory;
+import net.opengis.wps10.ComplexDataType;
+import net.opengis.wps10.DataInputsType1;
+import net.opengis.wps10.DataType;
+import net.opengis.wps10.ExecuteType;
+import net.opengis.wps10.InputType;
+import net.opengis.wps10.Wps10Factory;
+
+/**
+ * Tests the JDBC based process status store with a single instance
+ * 
+ * @author Ian Turton
+ */
+public class JDBCStatusStoreTest extends AbstractProcessStoreTest {
+
+    private DataStore datastore;
+
+    JDBCStatusStore statusStore;
+
+    @Override
+    protected ProcessStatusStore buildStore() {
+        Properties props = new Properties();
+        String home = System.getenv("HOME");
+        File f = new File(home, ".geotools/postgis.properties");
```

Comment:
This should really be an online type test, e..g, it should be skipped, not fail, if the postgis.properties file is not there